eng/GenerateAnalyzerNuspec.targets

```diff
@@ -40,8 +40,13 @@
     </ItemGroup>
 
     <PropertyGroup Condition="'$(GenerateAnalyzerMdFile)' == 'true'">
-      <AnalyzerDocumentationFileDir Condition="'$(AnalyzerDocumentationFileDir)' == ''">$(RepoRoot)src\$(NuspecPackageId)</AnalyzerDocumentationFileDir>
```

Comment:
Did you want to revert these deletions? We still want to generate documentation files.